notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb

```diff
@@ -0,0 +1,754 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    print(len(tbl))
</pre><p>I would also have this print out what "17" means, e.g., <code>print("Number of systems meeting our selection criteria:\n", len(tbl))</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>